strict_deps/index.bzl

```diff
@@ -100,3 +113,12 @@ strict_deps_test = rule(
         ),
     },
 )
+
+
+def strict_deps_test(**kwargs):
+    kwargs['$will_fail'] = False
```

Comment:
I might be missing something, but what does the `$` do here?

Reply:
Removed per discussion.